modules/csi-secrets-store-provider-aws/main.tf

```diff
@@ -4,9 +4,10 @@ locals {
 }
 
 module "secrets_store_csi_driver" {
```

Comment:
lets just remove this - if the chart that we are wrapping does not bake in its dependencies, neither should we since it will become too confusing and problematic. we can always show in our documentation that you need to enable the following for the AWS provider:

```hcl
  enable_secrets_store_csi_driver              = true
  enable_secrets_store_csi_driver_provider_aws = true
```